.github/workflows/cypress.yml

```diff
@@ -0,0 +1,19 @@
+name: cypress
+on: [deployment_status]
+jobs:
+  e2e:
+    # only runs this job on successful deploy
+    if: github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dump GitHub context
```

Comment:
Would it be possible to add a comment explaining why this step is necessary?

Reply:
I read this is needed to get the `target_url`. But I just tried without this step, and it's still working. So I have removed this part.